apps/web/components/ui/Form/FormPasswordInput.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <SfInput :type="isPasswordVisible ? 'text' : 'password'">
+    <template #suffix>
+      <button type="button" @click="isPasswordVisible = !isPasswordVisible">
+        <component :is="isPasswordVisible ? SfIconVisibilityOff : SfIconVisibility" />
+      </button>
+    </template>
+  </SfInput>
+</template>
+
+<script lang="ts" setup>
+import { SfInput, SfIconVisibility, SfIconVisibilityOff } from '@storefront-ui/vue';
+
+const isPasswordVisible = ref(false);
```

Comment:
```suggestion
const isPasswordVisible = ref();
```

Reply:
I was thinking about it, but wanted to have the correct typing (boolean.
Do you think this would be better?
```suggestion
const isPasswordVisible = ref<boolean>();
```

Reply:
I think set typing would be better

Reply:
e2f01af6fb485ed2e25a41fd84e1190151d29af2